test/accounts.js

```diff
@@ -1,263 +1,195 @@
-var BN = require("bn.js");
-var Web3 = require("web3");
-var Ganache = require(process.env.TEST_BUILD
-  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
-  : "../index.js");
-var assert = require("assert");
+const assert = require("assert");
+const initializeTestProvider = require("./helpers/web3/initializeTestProvider");
 
-describe("Accounts", function() {
-  var expectedAddress = "0x604a95C9165Bc95aE016a5299dd7d400dDDBEa9A";
-  var mnemonic = "into trim cross then helmet popular suit hammer cart shrug oval student";
+describe("Accounts", async() => {
+  const expectedAddress = "0x604a95c9165bc95ae016a5299dd7d400dddbea9a";
```

Comment:
I wonder if removing the checksumming here has unintended consequences.

Reply:
Any objections to restoring `expectedAddress` back to case-sensitive form and removing the use of `toLowerCase()` for address comparisons?